src/store/reducers.ts

```diff
@@ -346,14 +346,17 @@ export const messagesReducer = handleActions(
 
 
 // FIXME Move all the below (potentially the fetchCount ones too) to their own file
+export type ViewModeKey = "last" | "viewer" | "editor";
+
 export interface SettingsType {
   locale: string;
   logLevel: LogLevel;
   ranWizrd: boolean;
   theme: "auto" | "dark" | "light";
   fontSize: number;
   viewSettings: {
-    viewMode: boolean;
+    defaultViewMode: ViewModeKey;
+    lastViewMode: boolean;
```

Comment:
It's not a big deal this time, as this is a "live setting" that changes all the time, but just FYI, if you rename a field you need to also add a data migration for it so to keep the old value. Again, NO NEED to do it this time as at worst it'll happen just once to a user before it's set again.

Doing a migration will also have fixed the need to have fallback values in ItemEditScreen.

Reply:
Ah, thanks! That's how it should be done, I was wondering.
I noticed you didn't have to provide default values, but I had the same issue with the fontFamily settings and it didn't come up.

Isn't it better to do it anyway to have default values for new settings?

Reply:
Yeah, it's definitely better as it will also assure the types are correct. I think there should be an example migration in there, but if there isn't, take a look at etesync-web, there's a migration there for sure.